packages/experimental/Avatar/ios/MSFAvatarViewManager.m

```diff
@@ -0,0 +1,80 @@
+#import <React/RCTComponent.h>
+#import <React/RCTViewManager.h>
+
+@import FluentUI;
+
+ @implementation RCTConvert (MSFAvatarView)
```

Comment:
I probably need one of these for each enum.  I don't know why this works as is :D 
Also, nit: that one space